05-geometry-operations.qmd

```diff
@@ -141,12 +141,12 @@ The centroids and points in surface are illustrated in @fig-centroid-pnt-on-surf
 #| fig-cap: Centroids (black) and points on surface red of New Zealand and Seine datasets.
 
 fig, axes = plt.subplots(ncols=2)
-nz.plot(ax=axes[0], color="white", edgecolor="lightgrey")
-nz_centroid.plot(ax=axes[0], color="None", edgecolor="black")
-nz_pos.plot(ax=axes[0], color="None", edgecolor="red")
-seine.plot(ax=axes[1], color="grey")
-seine_pos.plot(ax=axes[1], color="None", edgecolor="red")
-seine_centroid.plot(ax=axes[1], color="None", edgecolor="black");
+nz.plot(ax=axes[0], color='white', edgecolor='lightgrey')
```

Comment:
Is this a Python-specific style thing? Looks fine to me in any case.

Reply:
Changed this to be uniform throughout the book, using `'` instead of `"`. IMHO both are considered normal in Python

Reply:
Makes sense, good to be consistent. We could at some point have a style guide but not a priority...